addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SqlInjectionScanRule.java

```diff
@@ -980,6 +989,7 @@ && matchBodyPattern(msg1, errorPattern, sb)) {
                 String resBodyANDTrueStripped =
                         stripOffOriginalAndAttackParam(
                                 resBodyANDTrueUnstripped, origParamValue, sqlBooleanAndTrueValue);
+                String headerANDTrue = msg2.getResponseHeader().getHeader(HttpHeader.LOCATION);
```

Comment:
I know you're following the existing code here but you could name the new variables with proper camelCase. ex: `headerAndTrue`

Reply:
Same for other vars below as well.

Reply:
updated